pdc/settings.py

```diff
@@ -29,7 +29,7 @@
 SECRET_KEY = '3hm)=^*sowhxr%m)%_u3mk+!ncy=c)147xbevej%l_lcdogu#+'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
I don't think we need to make this change in settings.py. Generally we will override it in settings_local.py.